packages/block-library/src/page-list/block.json

```
@@ -51,6 +51,12 @@
 		},
 		"interactivity": {
 			"clientNavigation": true
+		},
+		"color": {
+			"text": true,
+			"background": true,
+			"link": true,
+			"gradients": true
```

```suggestion
			"gradients": true,
			"__experimentalDefaultControls": {
				"background": true,
				"text": true,
				"link": true
			}
```

Apologies that I missed this feedback in my last review 🙏 

When there is color support, text and background support are automatically adopted. I like the explicit opt-in here, though, so let's keep it.

We do need to tweak the default controls, however. There's some history on the logic behind exposing default color controls in https://github.com/WordPress/gutenberg/pull/38511.

The general thinking was:

> In general, the following considerations informed which controls should be default:
> 
> - Text and Background controls should be defaults if supported
> - Link color will in most cases be optional
> - If a block explicitly opts out of text color support but into link color support, make the link control a default instead of text
> - If a block heavily features links, the link color will also be made a default control in that case

Once we tweak the default controls, this should be good to go 🚢